apps/zero-runtime-next-app/next.config.js

```diff
@@ -5,12 +5,6 @@ const { experimental_extendTheme: extendTheme } = require('@mui/material/styles'
 
 const theme = extendTheme();
 
-theme.applyDarkStyles = function applyDarkStyles(obj) {
```

Comment:
@mnajdova it appears that this change caused a breaking change in the Avatar component. Since when our team started consuming 5.15.6 we started to observe this error:
"TypeError: t.applyDarkStyles is not a function"

Reply:
This should not happen as this deletion is not part of the core `@mui/material` package and the method has been added in material-ui [here](https://github.com/mui/material-ui/pull/40324/files#diff-b65d8f9db0565878ce536e48a00589e9da7131a487c0ba91f6485d68b4f41fdfR46).

Reply:
We experience same issue. Starting from version `@mui/material@5.15.6` the Avatar component crashes with the message `TypeError: t.applyDarkStyles is not a function`, or `TypeError: e.applyStyles is not a function` for the later versions. We had to downgrade to `@mui/material@5.15.5` in order to fix the problem.

Reply:
Is there any update on this issue ??